src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -606,13 +606,6 @@ public static string ProjectFileVersion4IsNotSupportedAnymore(string projectFile
 
          public static string UnableToUpdateParameterException(string parameterPath, string simulationName) => $"Unable to update parameter.\nParameter with path '{parameterPath}' not found in simulation '{simulationName}'.";
 
-         public static string ConstantParameterAlreadyExistsInContainer(string containerName, string parameterName) => $"Parameter '{parameterName}' already exists in '{containerName}' but is defined as a constant parameter.";
```

Comment:
removes all of those constants that were just developer errors but we don't do errors anymore. Now we throw exception